sklearn/linear_model/_ridge.py

```diff
@@ -1704,6 +1741,13 @@ class RidgeCV(MultiOutputMixin, RegressorMixin, _BaseRidgeCV):
         below). This flag is only compatible with ``cv=None`` (i.e. using
         Generalized Cross-Validation).
 
+    alpha_per_target : bool, default=False
+        Flag indicating whether to optimize the alpha value (picked from the
+        `alphas` parameter list) for each target separately. When set to
+        `True`, after fitting, the `alpha_` attribute will contain a value for
+        each target. When set to `False`, a single alpha is used for all
+        targets. This flag has no effect when there is only one target.
```

Comment:
Currently, the flag looks like it will change the shape of the attributes. (They are no longer floats). We can check for one target and have a test to make sure the attributes (`alpha_`, etc) are still floats.